test/integration/scenarios/faultTests/solution/update/verify/manifest_test.go

```diff
@@ -0,0 +1,353 @@
+		ComponentsToAdd []string
+
+		// PodsToVerify specifies the pods need to be running
+		PodsToVerify []string
+
+		// DeletedPodsToVerify specifies the pods need to be deleted
+		DeletedPodsToVerify []string
+	}
+)
+
+var (
+	// manifestTemplateFolder includes manifest templates with empty components to deploy
+	manifestTemplateFolder = "../manifestTemplates"
+	// testManifestsFolder includes temporary manifest files for each test run. set in .gitignore
+	testManifestsFolder = "../manifestForTestingOnly"
+)
+
+var (
+	// Manifest templates
+	containerManifestTemplates = []string{
+		fmt.Sprintf("%s/solution-container.yaml", manifestTemplateFolder),
+	}
+
+	manifestTemplates = []string{
+		fmt.Sprintf("%s/target.yaml", manifestTemplateFolder),
+		fmt.Sprintf("%s/solution.yaml", manifestTemplateFolder),
+		fmt.Sprintf("%s/instance.yaml", manifestTemplateFolder),
+	}
+
+	// Manifests to deploy
+	testManifests = []string{
+		fmt.Sprintf("%s/target.yaml", testManifestsFolder),
+		fmt.Sprintf("%s/solution.yaml", testManifestsFolder),
+		fmt.Sprintf("%s/instance.yaml", testManifestsFolder),
+	}
+
+	testCases = []TestCase{
+		{
+			Name:                "Initial Symphony Target Deployment with nginx ingress",
+			Target:              "target",
+			ComponentsToAdd:     []string{"nginx-ingress"},
+			PodsToVerify:        []string{"proxy-nginx-ingress-controller"},
+			DeletedPodsToVerify: []string{},
+		},
+		{
+			Name:                "Update Symphony Target to add redis",
+			Target:              "target",
+			ComponentsToAdd:     []string{"nginx-ingress", "redis"},
+			PodsToVerify:        []string{"proxy-nginx-ingress-controller", "target-runtime-faultupdatetarget"},
+			DeletedPodsToVerify: []string{},
+		},
+		{
+			Name:                "Update Symphony Solution to add bitnami nginx",
+			Target:              "solution",
+			ComponentsToAdd:     []string{"bitnami-nginx"},
+			PodsToVerify:        []string{"proxy-nginx-ingress-controller", "target-runtime-faultupdatetarget", "nginx"},
+			DeletedPodsToVerify: []string{},
+		},
+		{
+			Name:                "Update Symphony Solution to remove bitnami nginx and add prometheus",
+			Target:              "solution",
+			ComponentsToAdd:     []string{"prometheus-server"},
+			PodsToVerify:        []string{"proxy-nginx-ingress-controller", "target-runtime-faultupdatetarget", "faultupdateinstance"},
+			DeletedPodsToVerify: []string{"nginx"},
+		},
+		{
+			Name:                "Update Symphony Target to remove nginx ingress and redis",
+			Target:              "target",
+			ComponentsToAdd:     []string{},
+			PodsToVerify:        []string{},
+			DeletedPodsToVerify: []string{"proxy-nginx-ingress-controller", "target-runtime-faultupdatetarget"},
+		},
+	}
+)
+
+func TestScenario_Update_AllNamespaces(t *testing.T) {
+	namespace := "nondefault"
+	defer shellcmd.Command(fmt.Sprintf("rm -rf %s", testManifestsFolder)).Run()
+	if namespace != "default" {
```

Comment:
Since the namespace is set in line 114, line 116 will always get `true`.

Reply:
fixed